data/tx_index.json

```diff
@@ -0,0 +1,4 @@
+{
+	"ns_index": [0, 0, 0, 0],
+	"tx_index": [0, 0, 0, 0]
```

Comment:
I don't suppose we want a nonzero reference value to catch things like endianness, etc.

Reply:
Yes that's a good idea

Reply:
Nice looks good